frontend/src/components/Root/Root.style.ts

```diff
@@ -13,14 +13,12 @@ export const PageContainer = styled.div`
   height: auto;
   width: auto;
 `;
-PageContainer.displayName = 'PageContainer';
```

Comment:
These `displayName` declarations were to simplify the inspection of components using the React Dev Tools. Could you check the difference between prod and your review app to see if you didn’t break anything in that regard?

Reply:
Ok I will rollback this part